buildsuites.xml

```diff
@@ -271,7 +275,7 @@
             <resources refid="ShortHeatMapViewerTests" />
             <resources refid="ShortHierarchicalClusteringTests" />
             <resources refid="ShortTopHatTests" />
-            <resources refid="Shorttxt2odfTests" />
+            <!-- <resources refid="Shorttxt2odfTests" /> --> <!-- Removing this from IU. Assuming it is not installed there either -->

```

Comment:
Line 268 - this doesn't impact the running of this - but you don't technically need Jenkins for this - just some sort of github/gpunit integration point.